### ISSUE TITLE
Search not returning correct results

### ISSUE THREAD

**Issue by ducatdusk** [BookStackApp/BookStack#1308]
**Describe the bug**
The search function is not returning correct results. It appears to be simply listing the content of the installation. I can search for a nonsense word that definately does not exist and it will return the same 30 results as if I search for a term that I know exists. 

**Steps To Reproduce**
1. Perform a search.

**Expected behavior**
Return search results relevent to the keyword given

**Your Configuration (please complete the following information):**
 - Exact BookStack Version (Found in settings): BookStack v0.25.1 
 - PHP Version: PHP 7.2.15-0ubuntu0.18.04.1
 - Hosting Method (Nginx/Apache/Docker): Nginx

**Additional context**

This issue arises whether searching across the installation, or within a book. I've also noticed that when I search within a book that the filer options reset as soon as I click submit. There does not appear to be any relevent errors in any of the log files available to me. I have tried to update the search index with php artisan, but to no avail.

Other than that, it's brilliant!

**Comment by ssddanbrown** [comment]
Hi @ducatdusk,
Sorry to hear you're having problems with BookStack.

Could you visit the [demo instance](https://demo.bookstackapp.com/login?email=admin@example.com&password=password) and let me know if the demo BookStack instance behaves different to your own one, in regards to the issues above?

**Comment by ukcolumn** [comment]
I am experiencing exactly the same issue as detailed above. I can confirm the demo instance works as expected, and that my test install on MAMP also works as expected, so this is clearly something to do with my production installation.

I have tested on the production box with php 7.1, 7.3 and 7.4. Search behaviour exactly as described above in all cases.

Hosting on Ubuntu 18.04, nginx/1.14.0 (Ubuntu)

BookStack v0.29.2 

The OP says: "I've also noticed that when I search within a book that the filer options reset as soon as I click submit." I also see this, but I would add that the url bar is updated, e.g. /search?term={type%3Abookshelf} The resulting search results remain the same, though: simply listing the content of the installation.

Any suggestions how I might track this down much appreciated.
